server/server.go

```diff
@@ -31,8 +32,11 @@ func newCribbageServer(dbFactory persistence.DBFactory) *cribbageServer {
 	}
 }
 
-func (cs *cribbageServer) NewRouter() http.Handler {
-	router := gin.Default()
+func (cs *cribbageServer) addRESTRoutes(router *gin.Engine) {
```

Comment:
:cactus: the others are `addWasmHandlers` and `addReactHandlers`, so it would make sense to call this `addRESTHandlers`.